requirements.txt

```diff
@@ -28,12 +28,12 @@ scp==0.14.5
 semver==3.0.1
 tabulate==0.9.0
 tqdm==4.66.1
-urllib3<2.0
+urllib3<3.0
```

Comment:
@adriengentil maybe it didn't understand the concept.. :rofl:

Reply:
:man_facepalming:

Reply:
at least it's not a hard dependency anymore, so I guess pip will select the right version :crossed_fingers: